packages/dataclass-editor/src/components/dataclass/detail/DataClassDetailContent.tsx

```diff
@@ -0,0 +1,43 @@
+import { BasicCheckbox, BasicField, Button, Flex, Textarea, ToggleGroup, ToggleGroupItem } from '@axonivy/ui-components';
+import { useState } from 'react';
+import { useAppContext } from '../../../context/AppContext';
+import { isEntityClass } from '../data/dataclass-utils';
+import './DataClassDetailContent.css';
+
+export const DataClassDetailContent = () => {
+  const { dataClass } = useAppContext();
+  const [classType, setClassType] = useState(isEntityClass(dataClass) ? 'entity-class' : 'data-class');
+
+  const variant = (value: string) => (value === classType ? 'primary' : undefined);
+
+  return (
+    <Flex direction='column' gap={4} className='detail-content'>
+      <BasicField label='Class type'>
+        <ToggleGroup type='single' className='class-type-group' value={classType} onValueChange={setClassType}>
+          <ToggleGroupItem value='data-class' asChild>
+            <Button variant={variant('data-class')} size='large'>
+              Data Class
+            </Button>
+          </ToggleGroupItem>
+          <ToggleGroupItem value='business-data-class' asChild>
+            <Button variant={variant('business-data-class')} size='large'>
+              Business Data Class
+            </Button>
+          </ToggleGroupItem>
+          <ToggleGroupItem value='entity-class' asChild>
+            <Button variant={variant('entity-class')} size='large'>
+              Entity Class
+            </Button>
+          </ToggleGroupItem>
+        </ToggleGroup>
+      </BasicField>
+      <BasicField label='Description'>
+        <Textarea value={dataClass.comment} />
+      </BasicField>
+      <BasicField label='Annotations'>
+        <BasicCheckbox label='BusinessCaseData' checked={dataClass.isBusinessCaseData} />
```

Comment:
What's up with having the option to choose `Business Data Class` in the field `Class type`? Is it something different than this or should it be a replacement for choosing this annotation?

Reply:
No idea... Before there where only normal data classes and entity classes, but only the normal data classes had the business class flag... So maybe this should be the replacement for this? 
Maybe @ivy-cst or @ivy-bb are know the idea better :)